.gitignore

```diff
@@ -6,6 +6,9 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+# TypeScript output folder
+lib
+
```

Comment:
Add
`node_modules`
`*.tsbuildinfo`

Reply:
`node_modules` is already there on line 44
`*.tsbuildinfo` is already there on line 51